CIP-0062/README.md

```diff
@@ -284,7 +284,7 @@ returned to the dApp with the following methods.
 
 ## Governance API
 
-All methods (all but the signing functionality) should not require any user
+Except `signVotes`, no other method should require any user
```

Comment:
`.submitDelegation()` should also need the user to authorise signing and submission of delegation transaction. Also, dependent on wallet implementations, `.getVotingKey()` may need user interaction -- to authorize mnemonic exposure for voting key derivation.